packages/hpe_aruba_cx/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -37,8 +37,66 @@ processors:
         field: event.kind
         tag: lowercase_event_kind
         ignore_missing: true
-    
-    # ECMP Events
+
+    # CoPP Events (15xx)
```

Could we include a link to the docs page for each log type as well? That has proved very helpful in other integrations (panw comes to mind). I know the README has it, but it's nice when it's right next to the code.

```suggestion
    # CoPP Events (15xx)
    # https://www.arubanetworks.com/techdocs/AOS-CX/10.07/HTML/5200-8214/Content/events/COPP.htm
```